### PR TITLE
fix INSTALL for MacOS

### PR DESCRIPTION
**Pull request by PeterPetrik**
note that the latest txt2tags removed some flags and change the formatting, so I had to tweak it manually afterward a bit...

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -1148,722 +1145,103 @@ is required.
...
 
-  5.6. QGIS source
+  5.4. QGIS source
   ================
 
-Unzip the QGIS source tarball to a working folder of your choice
-(/usr/somewhere is not a good choice as it's hidden and requires root
-privileges).  If you are reading this from the source, you've already done
-this.
+Unzip or the QGIS source to a working folder of your choice.
```
Comment (agiudiceandrea) [inline INSTALL:1243]:
```suggestion
Unzip the QGIS source to a working folder of your choice.
```
A typo?

File: INSTALL [hunk 2]
```diff
@@ -1148,722 +1145,103 @@ is required.
...
-Binary installers are available for OS X, but they are not recommended
-(2.4 versions install in /usr instead of /usr/local, and 2.6+ versions are a
-strange application). Instead, download the source.
-NOTE: 2.8.5 is broken for detecting part of Qt.  Fixed in 2.8.6.
-Double-click the source tarball to unpack it, then cd to the source folder and:
-
-  ./bootstrap --docdir=/share/doc/CMake --mandir=/share/man
-  make -j [#cpus]
-  sudo make install
+After installation of XCode, install command line tools
```
Comment (3nids) [inline INSTALL:1180]: do you really need xcode? developer tools are enough no?

File: INSTALL [hunk 3]
```diff
@@ -1893,105 +1265,44 @@ gives it the system app folder icon).  If you have the correct permissions you
...
 below assume you are building into a ${HOME}/Applications directory.
 
-You have two interactive options for configuring the build: ccmake or run
-Terminal commands. ccmake is a curses interface inside Terminal for CMake and
-allows a tabular layout for viewing and editing ALL available QGIS source CMake
-options. To get started initially run the Terminal method.
-
 In a Terminal cd to the qgis source folder previously downloaded, then:
 
   mkdir build
```
Comment (3nids) [inline INSTALL:1270]: it is not recommended anymore to have the build dirs within the source.
```suggestion
  cd ..
  mkdir build
```

File: INSTALL [hunk 4]
```diff
@@ -1893,105 +1265,44 @@ gives it the system app folder icon).  If you have the correct permissions you
...
+  
+  QGIS_DEPS_VERSION=0.3.0;\
+  QT_VERSION=5.14.1;\
+  PATH=/opt/QGIS/qgis-deps-${QGIS_DEPS_VERSION}/stage/bin:$PATH;\
+  cmake \
+    -DCMAKE_INSTALL_PREFIX=~/Applications \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DQGIS_MAC_DEPS_DIR=/opt/QGIS/qgis-deps-${QGIS_DEPS_VERSION}/stage \
+    -DCMAKE_PREFIX_PATH=/opt/Qt/${QT_VERSION}/clang_64 \
+    ..
```
Comment (3nids) [inline INSTALL:1281]:
```suggestion
    ../QGIS
```
or PATH_TO_QGIS_SOURCE

File: INSTALL [hunk 5]
```diff
@@ -1893,105 +1265,44 @@ gives it the system app folder icon).  If you have the correct permissions you
...
 source files in one directory up.
 
+Note: Double check on the screen output that all libraries are picked from QGIS-Deps `/opt/QGIS`
+and not from system `/usr/lib` or Homebrew's `/usr/local/` or system Frameworks `/Library/Frameworks/`.
+Especially check Proj, GDAL, sqlite3 and Python paths.
+
 After the initial Terminal configure, you can use ccmake to make further changes:
 
   cd build
   ccmake ..
```
Comment (3nids) [inline INSTALL:1293]: maybe add it to the install list above (but no biggie I guess, if you prefer to keep the list minimal)

**Review by 3nids — APPROVED** [review]
nice strip down!
many thanks for handling this @PeterPetrik , you've done an amazing work!

File: INSTALL (hunk 5 above)
Comment (PeterPetrik) [reply INSTALL:1293]: isn't ccmake part of cmake package?

File: INSTALL (hunk 2 above)
Comment (PeterPetrik) [reply INSTALL:1180]: i am not sure if it works without and not sure if I want to remove XCode to test...

File: INSTALL (hunk 5 above)
Comment (3nids) [reply INSTALL:1293]: oops, indeed

File: INSTALL (hunk 2 above)
Comment (3nids) [reply INSTALL:1180]: I have never installed XCode until recently and had no issue before.

Comment (PeterPetrik) [reply INSTALL:1180]: lucky you

**Review by timlinux — COMMENTED** [review]
That looks great!

**Comment by Codym48** [comment]
I regenerated INSTALL.html as well using the 3.6 tag of txt2tags over in #36212

**Pull request merged**